data/transit/route/subway.json

```diff
@@ -23,10 +23,14 @@
     {
       "displayName": "Delhi Metro",
       "id": "delhimetro-907487",
-      "locationSet": {"include": ["001"]},
+      "locationSet": {"include": ["in"]},
       "tags": {
         "network": "Delhi Metro",
-        "operator": "Delhi Metro",
+        "network:wikidata": "Q271195",
+        "network:wikipedia": "hi:दिल्ली_मेट्रो_रेल",
```

Comment:
I think I'm right in thinking we should use the local language by default?

Reply:
> I think I'm right in thinking we should use the local language by default?

Yes, I agree 👍